src/main/java/org/ays/common/model/entity/BaseEntity.java

```diff
@@ -88,7 +88,7 @@ public void preUpdate() {
                 .map(Authentication::getPrincipal)
                 .filter(user -> !"anonymousUser".equals(user))
                 .map(Jwt.class::cast)
-                .map(jwt -> jwt.getClaim(AysTokenClaims.USER_EMAIL_ADDRESS.getValue()).toString())
+                .map(jwt -> jwt.getClaim(AysTokenClaims.USER_ID.getValue()).toString())
                 .orElse("AYS");
         this.updatedAt = LocalDateTime.now();
     }
```

Comment:
Var olan end to end testlerde kontrol edecek şekilde 1-2 test senaryosu ekleyebilir miyiz? User ID olduğuna bakmamızda fayda var ✅

Reply:
tabii ki. Bugun ekliyor olurum ✅